fileHub/js-app/app/services/api-service/api-service.js

```diff
@@ -28,22 +31,23 @@ export class ApiService {
 
     const responseBody = await response.json();
 
-    if (response.ok) {
-      return responseBody;
-    } else if ((response.status >= 400 && response.status <= 421) ||
-        (response.status >= 423 && response.status < 500)) {
+    if ((response.status >= 400 && response.status < 500)) {
       throw new ClientServerError(responseBody.message);
-    } else if (response.status === 500) {
-      throw new ServerError(responseBody.message);
     }
+    return responseBody.token;
   }
 
   /**
    *
+   * @typedef {Object} userData
```

Comment:
Let's always name types starting with upper case.
Also, this model can be use in `logIn()` parameters so it's better to have a separate class.